.changeset/few-turtles-crash.md

```diff
@@ -0,0 +1,6 @@
+---
+"@logto/cli": patch
```

Comment:
Should be minor? A new feature is added.

Reply:
ok np